### PR TITLE
[Doc] Update mvpose docs

### PR DESCRIPTION
**Pull request by jiaqiAA**
- Updated the config files for easier starting.

### CODE REVIEW

File: docs/en/getting_started.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Please refer to [data\_preparation.md](./dataset_preparation.md) for data prepar
...
 
 If you want to obtain keypoints3d, the body model is not necessary.
-If you want to infer SMPL as well, you can prepare the body\_model as follows.
+If you want to infer SMPL as well, you can prepare the body model as follows.
 
-- [SMPL](https://smpl.is.tue.mpg.de/) v1.0 is used in our experiments.
+- [SMPL](https://smpl.is.tue.mpg.de/) v1.0.0 is used in our experiments. Please register to get access to the downloads section.
   - Neutral model can be downloaded from [SMPLify](https://smplify.is.tue.mpg.de/).
   - All body models have to be renamed in `SMPL_{GENDER}.pkl` format. <br/>
     For example, `mv basicModel_neutral_lbs_10_207_0_v1.0.0.pkl SMPL_NEUTRAL.pkl`
```
Comment (wqyin) [inline docs/en/getting_started.md:27]: Shall we provide the full path here to give better idea about where to find and where to put the model: `mv DOWNLOAD_PATH/smplify_public/code/models/basicModel_neutral_lbs_10_207_0_v1.0.0.pkl XRMOCAP_PATH/xrmocap_data/body_models/smpl/SMPL_NEUTRAL.pkl`?

File: scripts/download_weight.sh [hunk 2]
```diff
@@ -1,7 +1,9 @@
-mkdir weight
+mkdir -p weight/mvpose
 cd weight
 wget https://download.openmmlab.com/mmdetection/v2.0/faster_rcnn/faster_rcnn_r50_fpn_1x_coco/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth
 wget https://download.openmmlab.com/mmpose/top_down/hrnet/hrnet_w48_coco_wholebody_384x288_dark-f5726563_20200918.pth
 wget https://download.openmmlab.com/mmtracking/mot/faster_rcnn/faster-rcnn_r50_fpn_4e_mot17-half-64ee2ed4.pth
 wget https://download.openmmlab.com/mmtracking/mot/reid/tracktor_reid_r50_iter25245-a452f51f.pth
+cd mvpose
 wget https://openxrlab-share.oss-cn-hongkong.aliyuncs.com/xrmocap/weight/resnet50_reid_camstyle.pth.tar
```
[the remark this answers is not in the document]
Comment (jiaqiAA) [reply scripts/download_weight.sh:8]: We didn't train the Camstyle model, and we don't know its version.

File: docs/en/getting_started.md (hunk 1 above)
Comment (jiaqiAA) [reply docs/en/getting_started.md:27]: Here, we could explain how to rename a file, and the file path is mentioned below in the file structure.

File: scripts/download_weight.sh (hunk 2 above)
[the remark this answers is not in the document]
Comment (jiaqiAA) [reply scripts/download_weight.sh]: Done.

**Review by wqyin — APPROVED** [review]

**Pull request merged**